fedn/fedn/clients/reducer/restservice.py

```diff
@@ -375,7 +376,8 @@ def network_graph():
                 graph = plot.make_netgraph_plot(df_edges, df_nodes)
                 return json.dumps(json_item(graph, "myplot"))
             except Exception:
```

Comment:
why is there a try clause here if nothing is being handled?

Reply:
Good point, vill fix this in some future PR.